packages/world-module-erc20-external-store/test/ERC20.t.sol

```diff
@@ -0,0 +1,204 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.24;
+
+import { Test } from "forge-std/Test.sol";
+import { StoreSwitch } from "@latticexyz/store/src/StoreSwitch.sol";
+import { GasReporter } from "@latticexyz/gas-report/src/GasReporter.sol";
+import { RESOURCE_TABLE } from "@latticexyz/store/src/storeResourceTypes.sol";
+import { ResourceId, ResourceIdLib } from "@latticexyz/store/src/ResourceId.sol";
+
+import { Store } from "../src/Store.sol";
```

Comment:
there's a `StoreMock` you could use from the store package that does this (but we'd need to make sure its exported in the npm package, like `MudTest` in world package)

Reply:
The StoreMock was missing several functions that are needed when using Store without a World, like `getStaticField()` and `getDynamicField()`

Reply:
doesn't look like we're using those methods anymore?

either way, would be good to add those to the mock